pkg/collector/upgrade/upgrade_test.go

```diff
@@ -48,10 +48,10 @@ func TestShouldUpgradeAllToLatestBasedOnUpgradeStrategy(t *testing.T) {
 			// prepare
 			nsn := types.NamespacedName{Name: "my-instance", Namespace: "default"}
 			existing := makeOtelcol(nsn)
-			existing.Status.Version = beginV
 			err := k8sClient.Create(context.Background(), &existing)
 			require.NoError(t, err)
 
+			existing.Status.Version = beginV
```

Comment:
What's the story behind this change?

Reply:
The story is that after upgrading the dependencies the test was not working. Apparently the create call was reverting the version to empty string.

Reply:
Alright, so, a bug in the test?